core/src/avm2/globals/flash/display/NativeMenuItem.as

```diff
@@ -1,6 +1,33 @@
 package flash.display {
     import flash.events.EventDispatcher;
+    import __ruffle__.stub_constructor;
+    import __ruffle__.stub_getter;
+    // According to the documentation, it should be [API("661")]
+    // but airglobal.swc disagrees with that:
+    [API("667")]
     public class NativeMenuItem extends EventDispatcher {
```

Comment:
in FP, ContextMenu(Item) derives from NativeMenu(Item). Do we not want to do it here, or would it just complicate the current code too much?

Reply:
I have a confuse.
We already have `class ContextMenuItem extends NativeMenuItem` and `class ContextMenu extends NativeMenu`. Do you mean that this version gate could cause problems for there?

Reply:
It shouldn't, playerglobal gets the highest versions